labextension/package.json

```diff
@@ -1,40 +1,43 @@
 {
   "private": true,
   "name": "jupyterlab_geojson_labextension",
-  "version": "0.18.0",
+  "version": "0.20.2",
   "description": "A JupyterLab extension for rendering GeoJSON",
   "author": "Grant Nestor <grantnestor@gmail.com>",
-  "main": "src/plugin.js",
+  "main": "lib/plugin.js",
+  "files": [
+    "lib/*.js",
+    "*.css"
+  ],
   "keywords": [
     "jupyter",
     "jupyterlab",
     "jupyterlab extension"
   ],
+  "jupyterlab": {
+    "extension": true
+  },
   "scripts": {
-    "build": "node build_extension.js",
-    "watch": "watch \"npm install\" src ../component --wait 10 --ignoreDotFiles",
-    "preinstall": "npm install ../component",
-    "prepublish": "npm run build",
-    "extension:install": "jupyter labextension install --symlink --py --sys-prefix jupyterlab_geojson",
-    "extension:uninstall": "jupyter labextension uninstall --py --sys-prefix jupyterlab_geojson",
-    "extension:enable": "jupyter labextension enable --py --sys-prefix jupyterlab_geojson",
-    "extension:disable": "jupyter labextension disable --py --sys-prefix jupyterlab_geojson"
+    "build": "babel src --out-dir lib",
+    "prepublishOnly": "npm run build",
+    "watch": "watch \"npm run build\" src --wait 10 --ignoreDotFiles",
+    "extension:install": "jupyter labextension link .",
+    "extension:uninstall": "jupyter labextension unlink ."
   },
   "dependencies": {
-    "@jupyterlab/apputils": "^0.1.3",
-    "@jupyterlab/codemirror": "^0.1.3",
-    "@jupyterlab/docregistry": "^0.1.3",
-    "@jupyterlab/rendermime": "^0.1.3",
-    "@phosphor/algorithm": "^0.1.1",
-    "@phosphor/virtualdom": "^0.1.1",
-    "@phosphor/widgets": "^0.3.0",
+    "@jupyterlab/apputils": "^0.3.1",
+    "@jupyterlab/codemirror": "^0.3.1",
+    "@jupyterlab/docregistry": "^0.3.1",
+    "@jupyterlab/rendermime": "^0.3.1",
+    "@phosphor/algorithm": "^1.0.0",
+    "@phosphor/widgets": "^1.0.0",
+    "jupyterlab_geojson_react": "file:../component",
```

Comment:
Sorry, not sure I understand. Doesn't the build step happen after the ```npm pack``` and ```npm install```, at which point the ```component``` folder will not be available.